<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainwindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>751</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Cajero</string>
     </attribute>
     <widget class="QListWidget" name="listaMenuProductos">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>281</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="listaOrdenActual">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>20</y>
        <width>221</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditCliente">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>160</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Nombre del Cliente</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboDescuentos">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>200</y>
        <width>161</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAnadirItem">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>40</y>
        <width>101</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>AÃ±adir Item -&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnFinalizarPedido">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>240</y>
        <width>121</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Finalizar Pedido</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Cocinero</string>
     </attribute>
     <widget class="QListWidget" name="listaColaPedidos">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>381</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnProcesarSiguientePedido">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>30</y>
        <width>88</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Procesar</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>763</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
